gybitg/Views/Athlete/Index.cshtml

```diff
@@ -1,77 +1,234 @@
             </td>
+            <td></td>
+            <td></td>
+            <td text style="text-align:right;">
+                @Html.DisplayFor(model => model.RPG)
+            </td>
+        </tr>
+        <tr>
             <td>
-                @Html.DisplayFor(modelItem => item.HighschoolCoach)
+                @Html.DisplayNameFor(model => model.APG)
+            </td>
+            <td></td>
+            <td></td>
+            <td text style="text-align:right;">
+                @Html.DisplayFor(model => model.APG)
             </td>
+        </tr>
+    </table>
+    <h5 text style="color: darkorange;">Stats</h5>
+    <table>
+
+        <tr>
             <td>
-                @Html.DisplayFor(modelItem => item.HSGraduationDate)
+                @Html.DisplayNameFor(model => model.GP)
+            </td>
+            <td></td>
+            <td></td>
+            <td text style="text-align:right;">
+                @Html.DisplayFor(model => model.GP)
             </td>
+        </tr>
+        <tr>
             <td>
-                @Html.DisplayFor(modelItem => item.AAUId)
+                @Html.DisplayNameFor(model => model.GS)
+            </td>
+            <td></td>
+            <td></td>
+            <td text style="text-align:right;">
+                @Html.DisplayFor(model => model.GS)
             </td>
+        </tr>
+        <tr>
             <td>
-                <a asp-action="Edit" asp-route-id="@item.UserId">Edit</a> |
-                <a asp-action="Details" asp-route-id="@item.UserId">Details</a> |
-                <a asp-action="Delete" asp-route-id="@item.UserId">Delete</a>
+                @Html.DisplayNameFor(model => model.MPG)
+            </td>
+            <td></td>
+            <td></td>
+            <td text style="text-align:right;">
+                @Html.DisplayFor(model => model.MPG)
             </td>
         </tr>
-}
-    </tbody>
-</table>
+        <tr>
+            <td>
+                @Html.DisplayNameFor(model => model.MPG)
+            </td>
+            <td></td>
+            <td></td>
+            <td text style="text-align:right;">
+                @Html.DisplayFor(model => model.MPG)
+            </td>
+        </tr>
+        <tr>
+            <td>
+                @Html.DisplayNameFor(model => model.FGAG)
+            </td>
+            <td></td>
+            <td></td>
+            <td text style="text-align:right;">
+                @Html.DisplayFor(model => model.FGAG)
+            </td>
+        </tr>
+        <tr>
+            <td>
+                @Html.DisplayNameFor(model => model.FGMG)
+            </td>
+            <td></td>
+            <td></td>
+            <td text style="text-align:right;">
+                @Html.DisplayFor(model => model.FGMG)
+            </td>
+        </tr>
+
+    </table>
+    <h3>Contact</h3>
+
+</div>
+
+
+<br />
+<h1 style="text-align:center"> GALLERY </h1>
+<br />
+<div class="col-md-4">
+
+    <div class="video-container-gallery">
```

Comment:
Refactor as a for-each loop through your GalleryVideo collection

Reply:
Yes that makes sense.